src/hooks/api/ownerships.ts

```diff
@@ -44,22 +45,44 @@ const getOwnershipRequests = async ({ headers, organizerId }) => {
 
 type UseGetOwnershipRequestsArguments = ServerSideQueryOptions & {
   organizerId: string;
+  ownerId?: string;
 };
 
 const useGetOwnershipRequestsQuery = (
-  { req, queryClient, organizerId }: UseGetOwnershipRequestsArguments,
+  { req, queryClient, organizerId, ownerId }: UseGetOwnershipRequestsArguments,
   configuration: UseQueryOptions = {},
 ) =>
   useAuthenticatedQuery<OwnershipRequest[]>({
     req,
     queryClient,
     queryKey: ['ownership-requests'],
     queryFn: getOwnershipRequests,
-    queryArguments: { organizerId },
+    queryArguments: { organizerId, ...(ownerId && { ownerId }) },
     refetchOnWindowFocus: false,
     ...configuration,
   });
 
+const requestOwnership = async ({ headers, itemId, ownerId }) =>
+  fetchFromApi({
+    path: '/ownerships',
+    options: {
+      method: 'POST',
+      headers,
+      body: JSON.stringify({
+        itemId,
+        itemType: 'organizer',
+        ownerId,
+      }),
+    },
+  });
```

Comment:
I had to add the same request on my end in https://github.com/cultuurnet/udb3-frontend/pull/938 but in the actual page it has to work with `ownerEmail`. I think one of us will have to merge this back to make it handle both conditionally, perhaps by accepting both queryArguments but passin them conditionally in the body instead